### ISSUE TITLE
Mag 425A

### ISSUE THREAD

**Issue by slashsd1515** [SlaSerX/Ministra#7]
hello please i install the platform of ministra last one but the box mag425A is not uploading the portal  what can be the issue , all others works well

**Comment by SlaSerX** [comment]
check your settings on portal

**Comment by slashsd1515** [comment]
normally every think ok , when i test on mag or stb emu it works but on ministra client not ...

**Comment by slashsd1515** [comment]
plathform 5.6.0 and mag box 425A with last updat of ministra client

**Comment by SlaSerX** [comment]
U add support for this mag in config ?

На сб, 23.01.2021 г., 20:44 ч. slashsd1515 <notifications@github.com>
написа:

**Comment by slashsd1515** [comment]
need just to add supporting in models   , this mag 425A? just one ligne or there is other place ?

**Comment by SlaSerX** [comment]
yes

**Comment by slashsd1515** [comment]
yes we did , please any chance to get your support in live , by connecting on our server ,if you can ofcourse, my email is habes.khalil@gmail.com  ,or whatsapp as it is ok for you , thank you in advance

**Comment by SlaSerX** [comment]
add me on skype: slaserx@hotmail.com
